docs/config/README.md

```diff
@@ -25,26 +25,36 @@ $ tree ~/.minio
 │   └── public.crt
 ```
 
-### Configuration Fields
-#### Editing Configuration Fields
-For releases after minio.RELEASE.2017-08-05T00-00-53Z, configuration changes can be done using [mc]((https://docs.minio.io/docs/minio-client-quickstart-guide))
+### Accessing configuration file
+
+All configuration changes can be made using [`mc admin config` get/set commands](https://github.com/minio/mc/blob/master/docs/minio-admin-complete-guide.md). Following sections provide brief explanation of fields and how to customize them. A complete example of `config.json` is available [here](https://raw.githubusercontent.com/minio/minio/master/docs/config/config.sample.json)
+
+#### Editing configuration file fields
+
+Configuration changes should be done using [mc]((https://docs.minio.io/docs/minio-client-quickstart-guide))
+
 ##### Get current configuration for Minio deployment
+
 ```sh
 $ mc admin config get myminio/ > /tmp/myconfig
 ```
+
 ##### Set current configuration for Minio deployment
+
 ```sh
 $ mc admin config set myminio < /tmp/myconfig
 ```
 
 The `mc admin` config API will evolve soon to be able to configure specific fields using get/set commands.
 
 #### Version
+
 |Field|Type|Description|
 |:---|:---|:---|
 |``version``|_string_| `version` determines the configuration file format. Any older version will be automatically be migrated to the latest version upon startup. [DO NOT EDIT THIS FIELD MANUALLY]|
 
```

Comment:
s/Any older version will be automatically be/Any older version will automatically be/

Reply:
done